compiler/rustc_codegen_cranelift/src/constant.rs

```diff
@@ -175,6 +176,7 @@ pub(crate) fn codegen_const_value<'tcx>(
                 let mut alloc = Allocation::from_bytes(
                     std::iter::repeat(0).take(size.bytes_usize()).collect::<Vec<u8>>(),
                     align,
+                    Mutability::Not,
```

Comment:
@bjorn3 on the last PR I developed, `./x.py check compiler/rustc` checked the cranelift files -- but now that does not seem to be the case any more, I only got the cranelift build errors from PR CI.

Reply:
Are you sure you previously used `./x.py check compiler/rustc`? That explicitly says that `compiler/rustc` and its dependencies should be checked. cg_clif is not a dependency. `./x.py check compiler/rustc compiler/rustc_codegen_cranelift` should work though. I think `compiler/rustc` is even unnecessary as it is marked as dep of cg_clif.

Reply:
Yeah I am pretty sure -- I had that set in my vscode config.

Reply:
(and I think I saw the errors in vscode... though there is a slim chance that I am hallucinating this and actually did `./x.py check` on the console and copy-pasted the paths into vscode. I think I stopped doing that a while ago but maybe I misremember the timeline here.)